ompi/mca/pml/ob1/pml_ob1.c

```diff
@@ -826,6 +831,21 @@ void mca_pml_ob1_error_handler(
         return;
     }
 
+#if OPAL_ENABLE_FT_MPI
+    opal_output_verbose( 1, mca_pml_ob1_output,
+                         "PML:OB1: the error handler was invoked by the %s BTL for proc %s with info %s",
+                         btl->btl_component->btl_version.mca_component_name,
+                         (NULL == errproc ? "null" : OMPI_NAME_PRINT(&errproc->proc_name)), btlinfo);
+#if 0
```

Comment:
Should this be kept in the PR?

Reply:
This is very useful for debugging, but yeah. I'll remove it.. .